core/src/main/java/fr/sncf/osrd/standalone_sim/result/ResultTrain.java

```diff
@@ -75,18 +75,23 @@ public static final class SpacingRequirement {
         @Json(name = "end_time")
         public double endTime;
 
-        public SpacingRequirement(String zone, double beginTime, double endTime) {
+        // whether the requirement end_time is final. it's metadata, and **shouldn't be used for conflict detection**
+        @Json(name = "is_complete")
+        public boolean isComplete;
+
+        public SpacingRequirement(String zone, double beginTime, double endTime, boolean isComplete) {
             this.zone = zone;
             this.beginTime = beginTime;
             this.endTime = endTime;
+            this.isComplete = isComplete;
```

Comment:
Do we really want this to be in the editoast facing API?

Reply:
probably not

Reply:
@eckter can you make the field transient?